auxx/src/Command/Proc.hs

```diff
@@ -0,0 +1,468 @@
+    },
+
+    CommandProc
+    { cpName = "bvd"
+    , cpArgumentConsumer = do pure ()
+    , cpExec = \() -> ValueBlockVersionData <$> gsAdoptedBVData
+    , cpHelp = "return current (adopted) BlockVersionData"
+    },
+
+    CommandProc
+    { cpName = "send-to-all-genesis"
+    , cpArgumentConsumer = do
+        stagpDuration <- getArg tyInt "dur"
+        stagpConc <- getArg tyInt "conc"
+        stagpDelay <- getArg tyInt "delay"
+        stagpMode <- getArg tySendMode "mode"
+        stagpTpsSentFile <- getArg tyFilePath "file"
+        return Tx.SendToAllGenesisParams{..}
+    , cpExec = \stagp -> do
+        Tx.sendToAllGenesis sendActions stagp
+        return ValueUnit
+    , cpHelp = "create and send transactions from all genesis addresses \
+               \ for <duration> seconds, <delay> in ms. <conc> is the \
+               \ number of threads that send transactions concurrently. \
+               \ <mode> is either 'neighbours', 'round-robin', or 'send-random'"
+    },
+
+    CommandProc
+    { cpName = "send-from-file"
+    , cpArgumentConsumer = getArg tyFilePath "file"
+    , cpExec = \filePath -> do
+        Tx.sendTxsFromFile sendActions filePath
+        return ValueUnit
+    , cpHelp = ""
+    },
+
+    CommandProc
+    { cpName = "send"
+    , cpArgumentConsumer =
+        (,) <$> getArg tyInt "i"
+            <*> getArgSome tyTxOut "out"
+    , cpExec = \(i, outputs) -> do
+        Tx.send sendActions i outputs
+        return ValueUnit
+    , cpHelp = "send from #i to specified transaction outputs \
+               \ (use 'tx-out' to build them)"
+    },
+
+    CommandProc
+    { cpName = "vote"
+    , cpArgumentConsumer =
+        (,,) <$> getArg tyInt "i"
+             <*> getArg tyBool "agree"
+             <*> getArg tyHash "up-id"
+    , cpExec = \(i, decision, upId) -> do
+        Update.vote sendActions i decision upId
+        return ValueUnit
+    , cpHelp = "send vote for update proposal <up-id> and \
+               \ decision <agree> ('true' or 'false'), \
+               \ using secret key #i"
+    },
+
+    CommandProc
+    { cpName = "bvm"
+    , cpArgumentConsumer = do
+        bvmScriptVersion <- getArgOpt tyScriptVersion "script-version"
+        bvmSlotDuration <- getArgOpt tySecond "slot-duration"
+        bvmMaxBlockSize <- getArgOpt tyByte "max-block-size"
+        bvmMaxHeaderSize <- getArgOpt tyByte "max-header-size"
+        bvmMaxTxSize <- getArgOpt tyByte "max-tx-size"
+        bvmMaxProposalSize <- getArgOpt tyByte "max-proposal-size"
+        bvmMpcThd <- getArgOpt tyCoinPortion "mpc-thd"
+        bvmHeavyDelThd <- getArgOpt tyCoinPortion "heavy-del-thd"
+        bvmUpdateVoteThd <- getArgOpt tyCoinPortion "update-vote-thd"
+        bvmUpdateProposalThd <- getArgOpt tyCoinPortion "update-proposal-thd"
+        let bvmUpdateImplicit = Nothing -- TODO
+        let bvmSoftforkRule = Nothing -- TODO
+        let bvmTxFeePolicy = Nothing -- TODO
```

Comment:
slojnaaa

Reply:
I decided to leave these as `TODO` for now because it wasn't possible to specify these parameters in the previous version, so it's something for a separate issue.